### PR TITLE
repl: initial basic documentation

### PR DESCRIPTION
**Pull request by MykolaGolubyev**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -102,6 +102,8 @@
         <module>webtau-browser</module>
         <module>webtau-http</module>
         <module>webtau-http-groovy</module>
+        <module>webtau-graphql</module>
```
Comment (tsiq-karold) [inline pom.xml:105]: I don't necessarily mind the reordering here but I'm curious what the logic is for the order we have ended up with.  It's not alphabetical nor does it seem to be grouped in any way.

File: webtau-docs/znai/REPL/experiments.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Read Eval Print Loop
+
+Webtau command line tool comes with `REPL` mode that stands for 'read-eval-print-loop'. REPL is an established way to
```
Comment (tsiq-karold) [inline webtau-docs/znai/REPL/experiments.md:3]:
```suggestion
Webtau command line tool comes with a `REPL` mode that stands for 'read-eval-print-loop'. REPL is an established way to 
```

File: webtau-docs/znai/REPL/experiments.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Setting Config Value
+
+Use `cfg.url = "http://url"` to set base url for experiments.
+
+:include-file: cli-tests/http-repl-cfg/out.txt {title: "setting cfg value"}
+
+# Preserving Browser Context
+
+In case of browser testing REPL preserves the context of an opened browser, so you can experiment
```
Comment (tsiq-karold) [inline webtau-docs/znai/REPL/experiments.md:29]:
```suggestion
During browser testing the REPL preserves the context of an opened browser, so you can experiment
```

File: webtau-docs/znai/REPL/experiments.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Read Eval Print Loop
+
+Webtau command line tool comes with `REPL` mode that stands for 'read-eval-print-loop'. REPL is an established way to 
+perform interactive execution of an API for the purpose of learning or experimenting. 
+REPL preserves the context of execution and each new command you run can rely on the established context.
```
Comment (tsiq-karold) [inline webtau-docs/znai/REPL/experiments.md:5]:
```suggestion
The REPL preserves the context of execution and each new command you run can rely on the established context.
```

File: webtau-docs/znai/REPL/experiments.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+To run webtau in REPL mode run
+
+:include-cli-command: webtau repl 
+
+:include-file: cli-tests/repl-context/out.txt {title: "REPL session"}
+
+# Experiment With API
+
+Use REPL to try out various apis like `http.`, `browser.`, `db.`, `cli.`, etc.
```
Comment (tsiq-karold) [inline webtau-docs/znai/REPL/experiments.md:17]:
```suggestion
Use the REPL to try out various apis like `http.`, `browser.`, `db.`, `cli.`, etc.
```

File: webtau-docs/znai/REPL/experiments.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+:include-file: cli-tests/http-repl-cfg/out.txt {title: "setting cfg value"}
+
+# Preserving Browser Context
+
+In case of browser testing REPL preserves the context of an opened browser, so you can experiment
+with css selectors and element actions without re-opening the browser and setting the right web app state.
+
+:include-file: cli-tests/browser-repl-open/out.txt {title: "opened browser stays"}
+
+Using element selectors provide additional information in REPL mode to help with exploration
```
Comment (tsiq-karold) [inline webtau-docs/znai/REPL/experiments.md:34]:
```suggestion
Using element selectors provides additional information in REPL mode to help with exploration
```

File: pom.xml (hunk 1 above)
Comment (MykolaGolubyev) [reply pom.xml:105]: at the bottom is all sort of shaded/dist/e2e modules, almost like private functions

**Pull request merged**